codecov.yml

```diff
@@ -13,4 +13,11 @@ coverage:
         informational: true 
     project:
       default:
-        informational: true 
\ No newline at end of file
+        informational: true 
+
+# The file that contains code coverage information may contain coverage for files that we do not want to keep track of. 
+# Example: We do not want to know the coverage for the Tests/ directory because that does not contain source code of our SDK. 
+# https://docs.codecov.com/docs/ignoring-paths
+ignore: 
+  - "Tests" # ignore code coverage of Test/ files 
+  - "**/*.generated.swift" # Ignore code coverage of generated files.
```

Comment:
Added some ignore blocks to make our test coverage data more accurate. These files are not relevant to test coverage so we should ignore them.